src/blake.ts

```diff
@@ -0,0 +1,266 @@
+const zo = Uint8Array.from([0x01]);
```

Comment:
Was there a problem in using external package for that?
Please add copyright notice on the top of the file if it's impossible to switch back to black-hash.